src/api/add.js

```diff
@@ -53,27 +53,54 @@ export async function add({
 
 async function addToIndex({ dir, gitdir, fs, filepath, index }) {
   // TODO: Should ignore UNLESS it's already in the index.
-  const ignored = await GitIgnoreManager.isIgnored({
-    fs,
-    dir,
-    gitdir,
-    filepath,
+  filepath = Array.isArray(filepath) ? filepath : [filepath]
+  const promises = filepath.map(async filePathIterator => {
```

Comment:
filePathIterator it's a string, why do you call it `iterator`? I would rename it to `filepath` it will shadow the main parameter that is not needed in this context.

Reply:
I changed it to `currentFilepath` for clarity.  Not a fan of shadow.